src/pbt/utils/project_config.py

```diff
@@ -514,18 +524,34 @@ def __init__(
         self.fabric_config_without_conf_replace = copy.deepcopy(fabric_config)
         self.fabric_config = fabric_config.resolve_env_vars()
 
+    def get_db_base_path(self, fabric_id: Optional[str]):
+        if fabric_id is None:
+            return self.system_config.get_dbfs_base_path(None)
+        elif fabric_id and self.fabric_config.is_databricks_fabric(fabric_id):
+            volume_opt = self.fabric_config.get_fabric(fabric_id).databricks.volume
+            return self.system_config.get_dbfs_base_path(volume_opt)
+        return None
+
+    # fabric and DB connections are loosely linked and generally a multiple fabrics can point to a single DB workspace.
+    # so this information cannot be at DB layer it has to be at fabric layer.
+    def is_volume_supported(self, fabric_id):
+        if self.fabric_config.is_databricks_fabric(fabric_id):
+            volume_opt = self.fabric_config.get_fabric(fabric_id).databricks.volume
+            return volume_opt is not None
```

Comment:
this should also have a check that the path is not a normal (non-volume) dbfs path. 
Also, maybe have graceful handling of empty-string paths as well?